vrk-kpa/suomidigi
# SUOPA-303 Social media links

To test: 
- `make fresh`
- Go to any article or core content and scroll to the bottom of the page. 
- See that the Social media links looks ok and works. They should be centered on mobile and on aligned right on desktop
